src/components/Error/Error.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, makeStyles, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+
+interface ErrorProps {
+  navigateBack: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  variant?: 'Error' | '404';
+}
+
+const useStyles = makeStyles(({ spacing }) => ({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+  },
+  errorType: {
+    display: 'flex',
+  },
+  boldText: {
+    fontWeight: 'bold',
```

Comment:
If we must bold, let's do it this way:
```suggestion
    fontWeight: typography.weight.bold,
```

Otherwise let's just try to stick to the typography variants already defined.

Reply:
changed to `typography.weight.bold`